themes/default/content/templates/kubernetes/helm/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Helm Chart on Kubernetes
+layout: template
+
+meta_desc: The Helm Chart template makes it easy to deploy a web application to a Kubernetes cluster with Pulumi and Helm.
+
+meta_image: meta.png
+
+card_desc: Deploy a web application to a Kubernetes cluster with Pulumi and Helm.
+
+template:
+  prefix: helm-kubernetes
+  dirname: my-k8s-app
+  languages:
+    - typescript
+    - python
+    - go
+    - csharp
+    - yaml
+
+cloud:
+  name: Kubernetes
+  slug: kubernetes
+
+---
+
+The Helm Chart template creates an infrastructure as code project in your favorite language that deploys a Helm chart to an existing cluster using Pulumi. It uses Pulumi's [Native Kubernetes Provider](/registry/packages/kubernetes) to create a new namespace in an existing Kubernetes cluster and a new [Helm chart](/registry/packages/kubernetes/api-docs/helm/v3) that deploys a web application. The template generates a complete infrastructure as code program, including sample application code, to give you a working project out of the box that you can customize easily and extend to suit your needs.
+
+![An architecture diagram of the Pulumi Helm Chart template](./architecture.png)
+
+## Using this template
+
+To use this template to deploy a Helm chart to a Kubernetes cluster, make sure you've already provisioned a [Kubernetes cluster](/templates/kubernetes), [installed Pulumi](/docs/get-started/install) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/), and [configured your kubeconfig file](/registry/packages/kubernetes/installation-configuration#setup). Then create a new [project](/docs/intro/concepts/project) using the template in your language of choice:
+
+{{< templates/pulumi-new >}}
+
+Follow the prompts to complete the new-project wizard. When it's done, you'll have a complete Pulumi project that's ready to deploy and configured with the most common settings. Feel free to inspect the code in {{< langfile >}} for a closer look.
+
+## Deploying the project
+
+The template requires no additional configuration. By default, it will install an Nginx ingress controller with Helm. Once the new project is created, you can deploy it immediately with [`pulumi up`](/docs/reference/cli/pulumi_up):
+
+```bash
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output](/docs/intro/concepts/stack#outputs) values:
+
+name
+: The name of your new Helm chart deployment.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources.
+
+## Customizing the project
+
+You can change the behavior of the Helm chart by passing values to it and an example is already included in the template. Projects created with the Helm Chart template expose the following [configuration](/docs/intro/concepts/config) settings:
```

Comment:
I'd suggest moving this first sentence into a separate paragraph after the last code block in this section (before the "Tidying up") header. 

Also, I'd reword it to something like: "Further customization of the Helm chart itself is accomplished by passing values to the Helm chart in your Pulumi code. An example of passing a few values to the Helm chart is already included in the template for easy reference." (or similar)